extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -175,11 +175,6 @@ static public function onGetPreferences( $user, &$defaultPreferences ) {
 			}
 		}
 
-		if ( isset( $defaultPreferences['htmlemails'] ) ) {
```

Comment:
Should we remove the whole statement or just a check (and always invoke the three instructions that were inside the statement body)?